content/posts/newsletter-014/index.md

```diff
@@ -123,6 +123,30 @@ Imgur screenshot [gallery].
 [Mimas]: https://github.com/est31/mimas
 [gallery]: https://imgur.com/a/vvo7len
 
+### [Nox Futura]
```

Comment:
The corresponding URL for this link is missing (might be a copy and paste from an earlier newsletter?)